portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase

```diff
@@ -18,6 +18,36 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Bug: LRAC-11360 | Automation ID: LRAC-11361 | Test Summary: Paginate property list"
+	@priority = "4"
+	test CanPaginatePropertyList {
+		var propertyList = "Property Name 1,Property Name 2,Property Name 3,Property Name 4,Property Name 5,Property Name 6";
+
+		task ("go to property and add 6 properties") {
+			ACNavigation.goToSettings();
+
+			for (var n : list "${propertyList}") {
+				ACSettings.goToProperties();
+
+				ACProperties.addNewProperty(propertyName = "${n}");
+			}
+		}
+
+		task ("Change the pagination to 5") {
+			ACSettings.goToProperties();
+
+			ACUtils.changePagination(itemsPerPage = "5");
+		}
+
+		task ("Click next page") {
+			ACUtils.checkPreviousNextButton(nextButton = "true");
+		}
+
+		task ("Check that the existing properties in the AC appear in the list") {
+			ACUtils.assertItemsName(itemList = "Property Name 1");
+		}
```

Comment:
Hey @calvinctvo, very nice work here. I think you could check that the first page has properties 1,2,3,4,5 and that property 6 does not appear, so then you switch to page 2 and verify that only property 6 appears and that the others not. What do you think?

Reply:
Hey @ViniLopes20 , I think that can work. But also, I ran the test and property name added in created order, so Property 1 would be in the bottom.